src/main/java/com/playtika/homework1/task4/FirstAndSecondPositions.java

```diff
@@ -0,0 +1,60 @@
+package com.playtika.homework1.task4;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class FirstAndSecondPositions {
+
+    public static void main(String[] args) {
+        String[] names = {"Elena", "Thomas", "Hamilton", "Suzie", "Phil", "Matt", "Alex", "Emma", "John", "James", "Jane", "Emily", "Daniel", "Neda", "Aaron", "Kate"};
+        int[] times = {341, 273, 278, 329, 445, 402, 388, 275, 243, 334, 412, 393, 299, 343, 317, 265};
+        allResults(names, times);
+        certainPosition(names, times);
+    }
+
+    private static void allResults(String[] names, int[] times) {
+        System.out.println("Полный список результатов: \n");
+        int[] copy = new int[times.length];
+        copy = times.clone();
+        Arrays.sort(copy);
+        for (int i = 0; i < 16; i++) {
+            for (int j = 0; j < times.length; j++) {
+                if (copy[i] == times[j]) {
+                    System.out.println((i + 1) + " место занимает " + names[j] + " с результатом " + times[j] + " сек.");
+                }
+            }
+        }
+    }
+
+    private static void certainPosition(String[] names, int[] times) {
+        System.out.print("\n\nВведите номер позиции которая вас интересует: \n");
+        Integer position = getPosition();
+        int[] copy = new int[times.length];
+        copy = times.clone();
+        Arrays.sort(copy);
+        if (position <= 16) {
```

Comment:
from where you get 16? change everywhere to `times.length`, moreover extract `times.length` to variable if it's used more than once in a method